omegaconf/resolvers/oc/__init__.py

```diff
@@ -92,10 +94,25 @@ def deprecated(
     return target_node
 
 
+def select(
```

Comment:
Oh -- something else I want to point out (though I don't feel strongly about it) is that it's possible to select a parent node / create interpolation cycles with `oc.select`. We might want to prevent them similar to interpolations.